jovian/__init__.py

```diff
@@ -19,23 +19,14 @@
 
 def commit(secret=False, nb_filename=None, files=[], capture_env=True,
            env_type='conda', notebook_id=None, create_new=None, artifacts=[]):
-    """Save the notebook, capture environment and upload to the cloud for sharing.
+    """Commits a Jupyter Notebook with its environment to Jovian.
 
-    In most cases, commit works well with the default arguments. It attempts to 
-    1. Save the Jupyter notebook
-    2. Upload the notebook (and additional scripts, CSVs etc.) to https://jvn.io 
-    3. Capture the python environment (using Anaconda or pip)
-    4. Upload the python environment to cloud
+    Saves the checkpoint of the notebook, capture the required dependencies from the python environment and uploads the notebook, env file, additional files like scripts, csv etc. to https://www.jvn.io . Capturing the python environment ensures that the notebook can be reproduced and 
+    executed easily using the ***{links to reprodue notebooks}.***
```

Comment:
typo in `reprodue`

Reply:
Pls. spellcheck all the text before merging.

Reply:
Did the spellcheck for all files and resolved the typos